### ISSUE TITLE
Repository name

### ISSUE THREAD

**Issue by ThingEngineer** [kike-canaries/esp32-max30105#1]
Curious why this repository is named "esp32-max30105" when it is running on an ESP8266 NOT and ESP32.

**Comment by hpsaturn** [comment]
Yes, but it is only a first tests that we did. For a serial error in this moment we did changed to esp8266. Please, for complete and functional firmware please review the next repo:

https://github.com/kike-canaries/esp32-hpma115s0

max30105 for now, is only in testing state. Thanks
